domains/duyhung.json

```diff
@@ -4,6 +4,6 @@
     "email": "hungwt334@gmail.com"
   },
   "record": {
-    "A": ["82.165.193.230"]
+    "A": ["109.228.48.107"]
```

Comment:
Can you show a preview of your website?

Reply:
Dear friend, is this enough?
http://demo.wowonder.com/

Reply:
> Dear friend, is this enough? [http://demo.wowonder.com](http://demo.wowonder.com/)

1. Making multiple prs won't get it merged
2. That doesn't seem to be your website, when `dig`ging, that website uses this ip `31.172.80.78`, which is different from yours, `109.228.48.107`